src/fuzz_introspector/analyses/function_call_analyser.py

```diff
@@ -0,0 +1,256 @@
+# Copyright 2022 Fuzz Introspector Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Analysis for function call coverage in the project"""
+
+import logging
+
+from typing import (
+    List,
+    Tuple,
+    Dict
+)
+
+from fuzz_introspector import analysis
+from fuzz_introspector import cfg_load
+from fuzz_introspector import html_helpers
+from fuzz_introspector import utils
+from fuzz_introspector.datatypes import (
+    project_profile,
+    fuzzer_profile,
+    function_profile
+)
+
+logger = logging.getLogger(name=__name__)
+
+
+class Analysis(analysis.AnalysisInterface):
+    def __init__(self) -> None:
+        pass
+
+    @staticmethod
+    def get_name():
+        return "FunctionCallAnalyser"
```

Comment:
I'm wondering if this is the right name. Can we come up with something that describes the plugin a bit more? "FunctionCallAnalyser" seems a bit too general?

Reply:
ThirdPartyAPICoverageAnalyser?